packages/core/src/login/webview/vue/login.vue

```diff
@@ -366,13 +366,16 @@ export default defineComponent({
         await this.emitUpdate('created')
     },
 
-    mounted() {
+    async mounted() {
         this.fetchRegions()
-        void this.updateExistingConnections()
+        await this.updateExistingConnections()
 
         // Reset gathered telemetry data each time we view the login page.
         // The webview panel is reset on each view of the login page by design.
         void client.resetStoredMetricMetadata()
```

Comment:
should this also be await'd ?

Reply:
`resetStoredMetricMetadata` is not an async function so await won't be required.

https://github.com/ivikash/aws-toolkit-vscode/blob/45ae5d97bd8c6f5aa876f94ce394048e5be25797/packages/core/src/login/webview/vue/backend.ts#L224-L226

Reply:
then why are we calling it with `void` ?

Reply:
Thats right, it shouldn't have void. I am updating it now :).